GRDB/Core/Database.swift

```diff
@@ -1237,8 +1237,29 @@ extension Database {
     ///         try db.usePassphrase("secret")
     ///     }
     public func usePassphrase(_ passphrase: String) throws {
-        let data = passphrase.data(using: .utf8)!
-        let code = data.withUnsafeBytes {
+        guard !passphrase.isEmpty, var data = passphrase.data(using: .utf8) else {
```

Comment:
I could check that previous GRDB versions would also throw a DatabaseError of code SQLITE_ERROR (1). So this check does not introduce any breaking change 👍

Reply:
Actually, why do check for empty strings, since you check for empty data? And why is the check for empty data necessary?

Reply:
I was frustrated that the SQLite Error 1 was happening and breaking an existing test.  But I agree...this doesn't add value and I'll remove it.